Automattic/crowdsignal-ui
# Update themes/blocks to improve responsiveness

## Summary

This PR adds a few style changes to blocks and themes, aiming to make projects more responsive on smaller screens.

Fixes Automattic/crowdsignal#450

## Testing Instructions
* Open or create a project
* Go to the project preview
* Change the screen size and check if the blocks fit well, especially on smaller screens
* Make sure to check all themes.
